CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v3.0.0
+* Breaking change 
+  By default feed will not bind http port on 8080 if `--ingress-port=XXXX` is not provided   
```

Comment:
Just to avoid the double-negative :)

```suggestion
  Feed will not bind port http 8080 by default, unless `--ingress-port=XXXX` is provided   
```